ui/main/src/app/business/store/lightcards/lightcards-store.ts

```diff
@@ -246,6 +263,18 @@ export class LightCardsStore {
             } else {
                 this.childCards.set(card.parentCardId, [card]);
             }
+
+            const parent = this.lightCards.get(card.parentCardId);
```

Comment:
parent could be null if child is received before parent cards (we have no guarantee of the receive order when loading cards) so you should check this case 
But if it arise , the processing of the ack/read should be done when receiving the card , it is already done for read card in your code line 239 but it has to be done as well for ack